### PR TITLE
OpenRGB 0.9 | update + 64-bit!

### PR DESCRIPTION
**Pull request by Macleykun**
- [x ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ x] This PR only modifies one (1) manifest
- [ x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/119213)

### CODE REVIEW

**Comment by stephengillie** [comment]
Manual Validation failed with:
>  > 2023-09-08 17:16:46.265 [CLI ] Starting installer selection.  
> 2023-09-08 17:16:46.265 [CLI ] Installer [X64,burn,Machine,] not applicable: Architecture was excluded by caller : X64  
> 2023-09-08 17:16:46.266 [CLI ] No suitable installer found for manifest Microsoft.VCRedist.2015+.x64 with version 14.38.32919.0  
> 2023-09-08 17:16:46.267 [CLI ] Terminating context: 0x8a150104 at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\DependenciesFlow.cpp:12b

File: manifests/c/CalcProgrammer1/OpenRGB/0.9/CalcProgrammer1.OpenRGB.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    PortableCommandAlias: openrgb
+  InstallerUrl: https://openrgb.org/releases/release_0.9/OpenRGB_0.9_Windows_32_b5f46e3.zip
+  InstallerSha256: 983CC04BC14FCB824E8075C23BD9A39642B632C84784510FF81DD8FDD84331B4
+- Architecture: x64
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: './OpenRGB Windows 64-bit/OpenRGB.exe'
+    PortableCommandAlias: openrgb
+  InstallerUrl: https://openrgb.org/releases/release_0.9/OpenRGB_0.9_Windows_64_b5f46e3.zip
+  InstallerSha256: 4A42DF973BF9E0694268993478F03A71DAFBF2DDBCB1512835B4BBABDC6DC6DE
```
Comment (stephengillie) [inline manifests/c/CalcProgrammer1/OpenRGB/0.9/CalcProgrammer1.OpenRGB.installer.yaml:27]:
```suggestion
Installers:
- Architecture: x86
  NestedInstallerType: portable
  NestedInstallerFiles:
  - RelativeFilePath: './OpenRGB Windows 32-bit/OpenRGB.exe'
    PortableCommandAlias: openrgb
  InstallerUrl: https://openrgb.org/releases/release_0.9/OpenRGB_0.9_Windows_32_b5f46e3.zip
  InstallerSha256: 983CC04BC14FCB824E8075C23BD9A39642B632C84784510FF81DD8FDD84331B4
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x64
- Architecture: x64
  NestedInstallerType: portable
  NestedInstallerFiles:
  - RelativeFilePath: './OpenRGB Windows 64-bit/OpenRGB.exe'
    PortableCommandAlias: openrgb
  InstallerUrl: https://openrgb.org/releases/release_0.9/OpenRGB_0.9_Windows_64_b5f46e3.zip
  InstallerSha256: 4A42DF973BF9E0694268993478F03A71DAFBF2DDBCB1512835B4BBABDC6DC6DE
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```

Comment (stephengillie) [reply manifests/c/CalcProgrammer1/OpenRGB/0.9/CalcProgrammer1.OpenRGB.installer.yaml:27]: Hi @Macelykun,

I believe that installing both frameworks might be causing the error above.

**Comment by Macleykun** [comment]
> Manual Validation failed with:
> 
> > > 2023-09-08 17:16:46.265 [CLI ] Starting installer selection.
> > > 2023-09-08 17:16:46.265 [CLI ] Installer [X64,burn,Machine,] not applicable: Architecture was excluded by caller : X64
> > > 2023-09-08 17:16:46.266 [CLI ] No suitable installer found for manifest Microsoft.VCRedist.2015+.x64 with version 14.38.32919.0
> > > 2023-09-08 17:16:46.267 [CLI ] Terminating context: 0x8a150104 at D:\a_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\DependenciesFlow.cpp:12b

Thanks for your review and suggestion! Makes also more sense :p also applied it to the 64-bit version of 0.8

**Comment by Macleykun** [comment]
@stephengillie this one would be similar to the previous version :)

File: manifests/c/CalcProgrammer1/OpenRGB/0.9/CalcProgrammer1.OpenRGB.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  Dependencies:
+    PackageDependencies:
+      - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```
Comment (stephengillie) [inline manifests/c/CalcProgrammer1/OpenRGB/0.9/CalcProgrammer1.OpenRGB.installer.yaml:29]:
```suggestion
      - PackageIdentifier: Microsoft.VCRedist.2015+.x86
- Architecture: x64
  NestedInstallerType: portable
  NestedInstallerFiles:
  - RelativeFilePath: './OpenRGB Windows 64-bit/OpenRGB.exe'
    PortableCommandAlias: openrgb
  InstallerUrl: https://openrgb.org/releases/release_0.9/OpenRGB_0.9_Windows_64_b5f46e3.zip
  InstallerSha256: 4A42DF973BF9E0694268993478F03A71DAFBF2DDBCB1512835B4BBABDC6DC6DE
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Comment by stephengillie** [comment]
Hi @Macleykun,

Looks like the dependencies got swapped since the previous version. :)

**Comment by Macleykun** [comment]
> Hi @Macleykun,
> 
> Looks like the dependencies got swapped since the previous version. :)

oof, yup you're right :P thanks for spotting that one!

**Review by stephengillie — APPROVED** [review]

**Pull request merged**